x-pack/plugin/core/src/main/resources/org/elasticsearch/xpack/core/ml/config_index_mappings.json

```diff
@@ -328,6 +346,9 @@
           }
         }
       },
+      "deleting" : {
+        "type" : "keyword"
```

Comment:
Ah, we have to make this a `keyword` has previously created indices auto created this field as a `keyword`. I initially thought it should be a boolean.

Reply:
Yes, ideally we would have mapped this as a `boolean` back in 6.6 when the config index mappings were first created.

But every user who has ever deleted a job will have this field mapped as a `keyword`, so it has to be `keyword` now until we decide to move everyone over to a config index v2.